bitnami/kafka/templates/kafka-provisioning.yaml

```diff
@@ -85,6 +86,9 @@ spec:
           args:
             - -ec
             - |
+              sed -e 's/KAFKA_CLIENT_KEY_PASSWORD/$(KAFKA_CLIENT_KEY_PASSWORD)/g' "{{ .Values.persistence.mountPath }}/config/client.properties" > "{{ $clientPropertiesPath }}"
+              sed -i -e 's/KAFKA_CLIENT_KEYSTORE_PASSWORD/$(KAFKA_CLIENT_KEYSTORE_PASSWORD)/g' "{{ $clientPropertiesPath }}"
+              sed -i -e 's/KAFKA_CLIENT_TRUSTSTORE_PASSWORD/$(KAFKA_CLIENT_TRUSTSTORE_PASSWORD)/g' "{{ $clientPropertiesPath }}"
```

Comment:
Create `client.properties` similar to how `server.properties` is configured (https://github.com/bitnami/bitnami-docker-kafka/blob/6156e8acc59fbe7acb581607e417fde448037c90/3/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh#L477-L502)
```suggestion
              . /opt/bitnami/scripts/libkafka.sh
              export CLIENT_CONF="${CLIENT_CONF:-/opt/bitnami/kafka/config/client.properties}"
              if [ ! -f "$CLIENT_CONF" ]; then
                touch $CLIENT_CONF

                kafka_common_conf_set "$CLIENT_CONF" security.protocol {{ include "kafka.listenerType" ( dict "protocol" .Values.auth.clientProtocol ) | quote }}
                kafka_common_conf_set "$CLIENT_CONF" ssl.keystore.type {{ upper .Values.provisioning.auth.tls.type | quote }}
                kafka_common_conf_set "$CLIENT_CONF" ssl.truststore.type {{ upper .Values.provisioning.auth.tls.type | quote }}
                ! is_empty_value "$KAFKA_CLIENT_KEY_PASSWORD" && kafka_common_conf_set "$CLIENT_CONF" ssl.key.password "$KAFKA_CLIENT_KEY_PASSWORD"
                {{- if eq (upper .Values.provisioning.auth.tls.type) "PEM" }}
                file_to_multiline_property() {
                    awk 'NR > 1{print line" \\"}{line=$0;}END{print $0" "}' <"${1:?missing file}"
                }
                kafka_common_conf_set "$CLIENT_CONF" ssl.keystore.key "$(file_to_multiline_property "/certs/{{ .Values.provisioning.auth.tls.key }}")"
                kafka_common_conf_set "$CLIENT_CONF" ssl.keystore.certificate.chain "$(file_to_multiline_property "/certs/{{ .Values.provisioning.auth.tls.caCert }}")"
                kafka_common_conf_set "$CLIENT_CONF" ssl.truststore.certificates "$(file_to_multiline_property "/certs/{{ .Values.provisioning.auth.tls.cert }}")"
                {{- else if eq (upper .Values.provisioning.auth.tls.type) "JKS" }}
                kafka_common_conf_set "$CLIENT_CONF" ssl.keystore.location "/certs/{{ .Values.provisioning.auth.tls.keystore }}"
                kafka_common_conf_set "$CLIENT_CONF" ssl.truststore.location "/certs/{{ .Values.provisioning.auth.tls.truststore }}"
                ! is_empty_value "$KAFKA_CLIENT_KEYSTORE_PASSWORD" && kafka_common_conf_set "$CLIENT_CONF" ssl.keystore.password "$KAFKA_CLIENT_KEYSTORE_PASSWORD"
                ! is_empty_value "$KAFKA_CLIENT_TRUSTSTORE_PASSWORD" && kafka_common_conf_set "$CLIENT_CONF" ssl.truststore.password "$KAFKA_CLIENT_TRUSTSTORE_PASSWORD"
                {{- end }}
              fi
```